### DISCUSSION TITLE
Percy just hangs sometimes

### DISCUSSION THREAD

**Discussion by thomasmattheussen** [percy/cli#796] [Support]
Hi,

I'm running @percy/cli 1.0.0-beta.75 and @percy/storybook 4.0.3 in a Docker with node 14 and chromium.

I run this command: 

```$ npx percy storybook ./storybook-snapshots-static```

But sometimes, it just hangs. Any idea what it could be, or how I could debug this? This issue happens a lot, but not all the time.
This may be of note, I don't know, but I'm running this 19 times in a row (with different folders and PERCY_TOKEN ofcourse) for different projects within the same account. (36 snapshots per project)

Also (but I don't think this has anything to with the above), I get this warning about an invalid config:

```
[percy] Invalid config:
[percy] - static: unknown property
[percy] - upload: unknown property
```

although when running

````
$ npx percy config:validate
```

I get no errors in the config:

```
[percy] Found config file: .percy.json
[percy] Using config:
{
  version: 2,
  snapshot: {
    widths: [
      1280
    ],
    enableJavaScript: true
  }
}
```

**Comment by thomasmattheussen** [comment]
I just experienced this again, this time I had the verbose flag set. This is the only log I get, after this it hangs indefinetely.

```
[percy:config] Found config file: .percy.json (1480ms)
[percy:config] Using config:
{
  version: 2,
  snapshot: {
    widths: [
      1280
    ],
    enableJavaScript: true
  }
} (38ms)
[percy:cli:storybook] Serving Storybook build: ./storybook-snapshots-static (51ms)
[percy:cli:storybook] Get Storybook version: http://localhost:43963/?path=/settings/about (66ms)
[percy:cli:storybook] Get Storybook stories: http://localhost:43963/iframe.html (1ms)
[percy:core:browser] Launching browser (0ms)
[percy:core:browser] Browser connected [1891]: HeadlessChrome/93.0.4577.82 (2380ms)
[percy:core:page] Page created (30ms)
[percy:core:page] Page created (8ms)
```

**Comment by Robdel12** [comment]
Hey @thomasmattheussen! This makes things pretty suspicious:

```
[percy] Invalid config:
[percy] - static: unknown property
[percy] - upload: unknown property
```

This can happen if there's a bad dependency resolution or stale `node_modules` file. Curious if deleting the `node_modules` folder and reinstalling stops the hanging from happening. If not, we'll have to dig deeper (and likely will need a reproduction to debug)

**Reply by thomasmattheussen** [reply]
I've fixed that in the mean time by indeed reinstalling from scratch (noticed other people here having the same issue). The hangs still happen though. 

So how exactly can I help you help me?

**Reply by Robdel12** [reply]
> So how exactly can I help you help me?

A reproducible example would be the easiest way for us to jump in and figure out what's going on.

**Reply by thomasmattheussen** [reply]
Thanks, but I can't do that :/ private company code etc... Also, I can't reproduce it myself reliably.
